html/changelogs/Duny.yml

```diff
@@ -1,2 +1,7 @@
 author: Duny
-changes: []
+changes: 
+  - tweak: Deff: Re-added the security checkpoint in medbay.
```

Comment:
You need to put these lines between "" if they have weird characters like : because it doesn't like them